src/components/expression/expression.tsx

```diff
@@ -35,10 +36,20 @@ const colorToClassNameMap = {
   danger: 'euiExpression--danger',
 };
 
+const textWrapToClassNameMap = {
+  'break-word': 'euiExpression-breakWord',
+  truncate: 'euiExpression-truncate',
```

Comment:
Reminder, in BEM, double dashes `--` are used for modifier classes where a single dash `-` is for state.
```suggestion
  truncate: 'euiExpression--truncate',
```